browser/ui/brave_browser_window.h

```diff
@@ -6,9 +6,7 @@
 #ifndef BRAVE_BROWSER_UI_BRAVE_BROWSER_WINDOW_H_
 #define BRAVE_BROWSER_UI_BRAVE_BROWSER_WINDOW_H_
 
-#include <map>
-#include <vector>
-
+#include "brave/browser/ui/commands/accelerator_service.h"
```

Comment:
I'm not sure if this file is used on Android and if we should guard this header..

Reply:
In `browser_window.h#42`:

```cpp
#if BUILDFLAG(IS_ANDROID)
#error This file should only be included on desktop.
#endif
```

so I think we're good.